lms/djangoapps/commerce/views.py

```diff
@@ -140,6 +141,14 @@ def post(self, request, *args, **kwargs):  # pylint: disable=unused-argument
         except (exceptions.SlumberBaseException, exceptions.Timeout) as ex:
             log.exception(ex.message)
             return InternalRequestErrorResponse(ex.message)
+        finally:
+            audit_log(
```

Comment:
I agree that it's ok.  we shouldn't be making assumptions about the correct ordering of messages across systems anyway.